src/CreatorToken.sol

```diff
@@ -32,6 +33,8 @@ contract CreatorToken is ERC721 {
   uint256 public immutable CREATOR_FEE_BIPS;
   uint256 public immutable ADMIN_FEE_BIPS;
   uint256 private constant MAX_FEE = 2500; // 25% in bips
+  uint256 private constant MIN_HOLDING_TIME = 60;
+  mapping(uint256 => uint256) public purchaseTime;
```

Comment:
Couple of things:

* I think my instinct here is to keep this mapping private, since it's purely an internal concern of the contract and not of use to users or integrators. What do you think?
* I think we can used name parameters for the mapping since we're on 0.8.20, so we should, for readability!

Reply:
Made it private and added named parameters here: [77d2ad4](https://github.com/showtime-xyz/creator-tokens/pull/38/commits/77d2ad4397a0e6a3979a7554cb64398d9a505ccf)